babel/src/board.js

```diff
@@ -231,12 +231,12 @@ export class Board {
 
         _.each(matches, match => {
             // log data
-            let [xx, yy] = match[0];
-            matchData.push([this.orbs[xx][yy], match.length]);
+            let [rowrow, colcol] = match[0];
```

Comment:
`_row` is a much better anonymous identifier here.

`rowrow` makes me think of a children's song.